src/dev/BQ76952.cpp

```diff
@@ -226,26 +226,45 @@ BQ76952::Status BQ76952::communicationStatus() {
     }
     return BQ76952::Status::ERROR;
 }
+// Create struct that will hold CAN interrupt parameters
 
-BQ76952::Status BQ76952::getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum) {
+
+BQ76952::Status BQ76952::getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t& sum, cellVoltageInfo& voltageInfo) {
     Status status = Status::OK;
     uint8_t cellVoltageReg = CELL_VOLTAGE_BASE_REG;
-
+    //Must use temporary storage variables or else the values displayed in PEAK will be inaccurate from regular changes.
     uint32_t currentVoltage = 0;
+    int16_t currentMinVoltage = 4;
+    int16_t currentMaxVoltage =0 ;
```

Comment:
It looks like the GitHub action didn't run on your latest commit, which is why the autoformatter didn't fix this. Be sure to get it running again before pulling.